pilot/pkg/bootstrap/server.go

```diff
@@ -181,6 +181,12 @@ func NewServer(args *PilotArgs) (*Server, error) {
 
 	prometheus.EnableHandlingTimeHistogram()
 
+	// Parse and validate Istiod Address.
+	host, port, err := s.parseIstiodAddress()
```

Comment:
My standard comment: please use less generic terms. There are dozens of ports we use, and same for host. It seems everything in istio is using the most generic term possible.

I guess this is the istiodGrpcPort/host ?